packages/features/bookings/lib/getBookingResponsesSchema.ts

```diff
@@ -111,7 +111,15 @@ function preprocess<T extends z.ZodType>({
       }
       for (const bookingField of bookingFields) {
         const value = responses[bookingField.name];
-        const stringSchema = z.string();
+        let stringSchema = z.string();
+        if (bookingField.type === "textarea") {
+          if (typeof bookingField.maxLength === "number") {
+            stringSchema = stringSchema.max(bookingField.maxLength);
+          }
+          if (typeof bookingField.minLength === "number") {
+            stringSchema = stringSchema.min(bookingField.minLength);
+          }
+        }
```

Comment:
We could simplify it like this. 
```suggestion
       const maxLength = bookingField.maxLength ?? 0;
       const minLength = bookingField.minLength ?? HARD_LIMIT_MAX_LENGTH;
       const stringSchema = z.string().min(minLength).max(maxLength)
```

Reply:
Also, we should be able to introduce `superRefine` for `textarea` type here https://github.com/calcom/cal.com/blob/e47103361e0c38a4c014ea526a724548fccbc183/packages/features/form-builder/schema.ts#L319 like we did for `name` type.